avm/res/kubernetes/connected-cluster/main.bicep

```
@@ -0,0 +1,163 @@
+metadata name = 'Kubernetes Connected Cluster'
+metadata description = 'This module deploys an Azure Arc connected cluster.'
+
+// ============== //
+//   Parameters   //
+// ============== //
+
+@description('Required. The name of the Azure Arc connected cluster.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+@description('Optional. The identity type for the cluster. Allowed values: "SystemAssigned", "None".')
+@allowed([
+  'SystemAssigned'
+  'None'
+])
+param identityType string = 'SystemAssigned'
+
+@description('Optional. Tags for the cluster resource.')
+param tags object?
+
+@description('Optional. Optional. The Azure AD tenant ID.')
+param aadTenantId string = ''
```

Instead of an empty default value please use nullable property
```suggestion
param aadTenantId string?
```